Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -7520,6 +7521,73 @@
         <Mover value="LarvaReleaseMissile"/>
         <EditorCategories value="ObjectType:Projectile,ObjectFamily:Melee"/>
     </CUnit>
+    <CUnit id="AP_Morphling">
+        <DeathRevealRadius value="3"/>
+        <Race value="Zerg"/>
+        <Mob value="Multiplayer"/>
+        <Attributes index="Light" value="1"/>
+        <Attributes index="Biological" value="1"/>
+        <Response value="Flee"/>
+        <LifeStart value="25"/>
+        <LifeMax value="25"/>
+        <LifeArmor value="10"/>
+        <LifeRegenRate value="0.2734"/>
+        <LifeArmorName value="Unit/LifeArmorName/ZergGroundArmor"/>
+        <ShieldArmorName value="Unit/ShieldArmorName/ZergWildMutation"/>
+        <Speed value="0.6"/>
+        <Acceleration value="1000"/>
+        <SpeedMultiplierCreep value="3"/>
+        <Sight value="5"/>
+        <Food value="-1"/>
+        <CostCategory value="Army"/>
+        <CostResource index="Minerals" value="100"/>
+        <CostResource index="Vespene" value="25"/>

```

Comment:
50 minerals, 0 vespene 0.5 supply
That's the cheapest advanced unit (Baneling with Rapid Metamorph)

Reply:
Banelings are spawned in pairs, so in testing it cost 5 vespene to morph them (100/30); I didn't know about rapid metamorph though. I'll remove the gas cost

Reply:
BTW Rapid Metamorph should also reduce morphing time from Morphling